src/app/projects/components/project-settings/components/embeddings/embeddings.component.ts

```diff
@@ -26,16 +30,28 @@ export class EmbeddingsComponent implements OnInit, OnDestroy, OnChanges {
   @Input() embeddingHandles: { [embeddingId: string]: any };
   @Input() dataHandlerHelper: DataHandlerHelper;
   @Input() attributes: Attribute[];
+  @Input() embeddingPlatforms: EmbeddingPlatform[];
 
+  @ViewChild('gdprText') gdprText: ElementRef;
   downloadedModels: DownloadedModel[];
   subscriptions$: Subscription[] = [];
   somethingLoading: boolean = false;
   downloadedModelsQuery$: any;
+  embeddingHandlesCopy: { [embeddingId: string]: any };;
+  selectedPlatform: EmbeddingPlatform;
+  organization: Organization;
 
-  constructor(private projectApolloService: ProjectApolloService) { }
+  get PlatformType(): typeof PlatformType {
+    return PlatformType;
+  }
+
+  constructor(private projectApolloService: ProjectApolloService, private organizationApolloService: OrganizationApolloService) { }
 
   ngOnInit(): void {
     this.prepareDownloadedModels();
+    this.organizationApolloService.getUserOrganization().pipe(first()).subscribe((organization) => {
+      this.organization = organization;
+    });
```

Comment:
when this is called only on init a change wont be visible.
A function that is called here + on websocket notification